lib/util/string_conv/string_conv.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
please move it somewhere to `zephyr/include`, maybe `include/sys`. I know that fnmatch has header in its own directory but it is rather an exception and i don't know what is the reason for that exception.

Reply:
Ok, I will do that